README.md

```diff
@@ -32,5 +32,11 @@ To perform it just do `docker-compose up -d` in your console
 
 ## Development mode
 
-* Reinitialize gradle project in your IDE to generate sources for Data Access Layer `dal` or build:
-`./gradlew build`
+### Applying Flyway migrations
+* Run `./gradlew flywayMigrate`
+
+### Generating sources for Data Access Layer
+* Run `./gradlew clean build -PjooqCodegen`
```

Comment:
why explicit `clean` task execution needed here?
Probably for obsolete generated code removal `clean` not works, need also to add custom `"cleanup"` step

Reply:
Looks like JOOQ stores some meta info in `out` folder, therefore `clean` itself does not remove the generated code but makes the generator to recreate the sources during the next `build` execution instead.

Reply:
tried on my machine.
1. after `./gradlew clean -PjooqCodegen` command execution all generated sources from `./dal/src/main/java` were removed. We need to inform developer about that. that all manually created sources in mentioned dir will be removed in codegen phase.
2. in some cases (e.g. table was dropped) `build` phase will fail (another code uses old generated sources and project will not compile). To avoid that, use specific goal for jooq plugin: `:jooq-codegen-primary` (`./gradlew clean :dal:jooq-codegen-primary -PjooqCodegen`)
3. script will be mostly used from console, then i suppose to keep `profile`'s name shorter (e.g. `jooq`)

Reply:
1. Right, missed that. `./gradlew clean` itself doesn't remove the files, but `./gradlew clean -PjooqCodegen` does indeed. Hopefully `./gradlew clean -PjooqCodegen` will only be used when developer is explicitly regenerating the JOOQ files, so it should be impossible to accidentally loose the files.
2. Agree, I will update the readme file
3. I had an idea to name the profile  `jooq` but decided `jooqCodegen` is more self-explanatory.  As for creating a new task, this will not be easy but I will try.